src/client.cpp

```diff
@@ -629,7 +629,8 @@ void Client::step(float dtime)
 						do_mapper_update = false;
 				}
 
-				if (r.mesh && r.mesh->getMesh()->getMeshBufferCount() == 0) {
+				if (r.mesh && r.mesh->getMesh(0)->getMeshBufferCount() == 0 &&
+						r.mesh->getMesh(1)->getMeshBufferCount() == 0) {
```

Comment:
`MAX_TILE_LAYERS`? well, the statement will grow a bit... `bool flag = !!r.mesh; for(...`